services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.dateparse import parse_date
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
```

Comment:
The type hint for `movie_show_time` is incorrect. It should be a `datetime` object, not `int | float`. Consider using `datetime` for accurate representation of date and time.